packages/codegen-ui-react/lib/__tests__/forms/validation.test.ts

```diff
@@ -157,77 +157,105 @@ describe('validateField tests', () => {
-    expect(
-      validateField(endTime1, [
-        { type: ValidationTypes.BE_AFTER, strValues: [startTime.toString()], validationMessage: 'test' },
-      ]),
-    ).toEqual({ hasError: true, errorMessage: 'test' });
-  });
-  it('should validate BE_BEFORE type', () => {
-    const startDate = new Date('2022-01-09').toString();
-    const endDate1 = new Date('2023-01-09').toString();
-    const endDate2 = new Date('2021-01-09').toString();
+      expect(timestamp.toString()).toEqual(time);
+      expect(new Date(invalidTimestamp).toString()).toEqual('Invalid Date');
+      expect(ddMMYYYY).toEqual('01 Jan 2022');
+      expect(mmDDYYYY).toEqual('1/01/2022');
+      expect(yyyyMMDD).toEqual('2022-1-1');
+    });
+    it('should validate BE_AFTER type', () => {
+      // timestamp tests
+      const startTime = Date.now();
+      const endTime1 = startTime + 10;
+      const endTime2 = startTime - 10;
 
-    expect(
-      validateField(startDate, [
-        { type: ValidationTypes.BE_BEFORE, strValues: [endDate1.toString()], validationMessage: '' },
-      ]),
-    ).toEqual({ hasError: false });
-    expect(
-      validateField(startDate, [
-        { type: ValidationTypes.BE_BEFORE, strValues: [endDate2.toString()], validationMessage: '' },
-      ]),
-    ).toEqual({ hasError: true, errorMessage: `The value must be before ${endDate2}` });
-    expect(
-      validateField(startDate, [
-        { type: ValidationTypes.BE_BEFORE, strValues: [endDate2.toString()], validationMessage: 'test' },
-      ]),
-    ).toEqual({ hasError: true, errorMessage: 'test' });
+      const dateFormatTestCases = [
+        {
+          startDate: '01 Jan 2022',
+          endDate1: 'Jan 08 2023',
+          endDate2: '01 Jan 2021',
+        },
+        {
+          startDate: '01/09/2022',
+          endDate1: '01/09/2023',
+          endDate2: '01/09/2021',
+        },
+        {
+          startDate: '2022-1-9',
+          endDate1: '123123asdfdf123123',
+          endDate2: '2021-1-9',
+        },
+        {
+          startDate: startTime,
+          endDate1: endTime1.toString(),
+          endDate2: endTime2.toString(),
+        },
+      ];
 
-    const startTime = Date.now();
-    const endTime1 = startTime + 10;
-    expect(
-      validateField(startTime, [
-        { type: ValidationTypes.BE_BEFORE, strValues: [endTime1.toString()], validationMessage: '' },
-      ]),
-    ).toEqual({ hasError: false });
-    expect(
-      validateField(endTime1, [
-        { type: ValidationTypes.BE_BEFORE, strValues: [startTime.toString()], validationMessage: '' },
-      ]),
-    ).toEqual({ hasError: true, errorMessage: `The value must be before ${startTime}` });
-    expect(
-      validateField(endTime1, [
-        { type: ValidationTypes.BE_BEFORE, strValues: [startTime.toString()], validationMessage: 'test' },
-      ]),
-    ).toEqual({ hasError: true, errorMessage: 'test' });
+      dateFormatTestCases.forEach(({ startDate, endDate1, endDate2 }) => {
```

Comment:
Use Kests built in `.each` rather than a `forEach`. It makes it more clear which case is failing

Reply:
Ah yes. Ill open another PR